tests/PluginTest.php

```diff
@@ -35,8 +35,28 @@ public function testGetSubscribedEvents()
         $this->assertInternalType('array', $plugin->getSubscribedEvents());
     }
 
+    /**
+    * Tests handleTableflipCommand().
+    */
     public function testHandleTableflipCommand()
     {
+        $event = $this->getMockCommandEvent();
+        Phake::when($event)->getCustomParams()->thenReturn(array());
+        Phake::when($event)->getSource()->thenReturn('#channel1');
+        Phake::when($event)->getCommand()->thenReturn('PRIVMSG');
+        $queue = $this->getMockEventQueue();
+
+        $plugin = new Plugin;
+        $plugin->handleTableflipCommand($event, $queue);
+
+        Phake::when($event)->getCustomParams()->thenReturn(array('#channel1', 'tableflip some text'));
+        $plugin->handleTableflipCommand($event, $queue);
+
+        Phake::inOrder(
```

Comment:
try this for line 56

Phake::verify($queue, Phake::atLeast(1))->ircPrivmsg('#channel1', isString('tableflip some text'))